### PR TITLE
removed 521 ecdsa algorithm (not supported anymore)

### PR DESCRIPTION
**Pull request by M-Ahadi**
#### Type of change

- Documentation update

#### Description
fixed 521 to 512 ecdsa algorithm

### CODE REVIEW

File: docs/source/users-guide.rst [hunk 1]
```diff
@@ -451,7 +451,7 @@ Elliptic Curve (ECDSA) offers the following key size options:
 +--------+--------------+-----------------------+
 | 384    | secp384r1    | ecdsa-with-SHA384     |
 +--------+--------------+-----------------------+
-| 521    | secp521r1    | ecdsa-with-SHA512     |
```
Comment (denyeart) [inline docs/source/users-guide.rst:454]: Although it looks strange, 521 is a real curve. @ale-linux can you confirm it is supposed to be 521?

Comment (ale-linux) [reply docs/source/users-guide.rst:454]: The curve is indeed called variations of P and 521 (which is the bitlength of the prime group order), not 512. That said: the curve is being phased out and we in fact no longer support it, so we can just go ahead and remove this row.

Comment (M-Ahadi) [reply docs/source/users-guide.rst:454]: @denyeart that line is removed from the document

**Comment by M-Ahadi** [comment]
@denyeart is there anything needed to be done for this merge request to be merged?

**Review by lindluni — APPROVED** [review]

**Comment by ryjones** [comment]
@dependabot rebase

**Comment by M-Ahadi** [comment]
@mastersingh24 please take a look at this merge request, and merge if it look good.

**Review by mbwhite — APPROVED** [review]

**Comment by mbwhite** [comment]
@ryjones  I've added an approval, but I don't have merge rights on this repo.

**Review by sykesm — APPROVED** [review]

**Pull request merged**